tests/storages_tests/test_with_server.py

```diff
@@ -115,7 +118,17 @@ def test_loaded_trials(storage_url: str) -> None:
 )
 def test_store_infinite_values(input: float, expected: float, storage_url: str) -> None:
 
-    storage = optuna.storages.RDBStorage(url=storage_url)
+    storage: Union[RDBStorage, RedisStorage]
+    # This way is a workaround. When optuna.storages.get_storage() supports raw
+    # storage mode, use it like this.
+    # storage = optuna.storages.get_storage(url=storage_url, raw=True)
```

Comment:
I think quite a few users want to get raw storage because it significantly increases the number of queries. So, I'm a bit conservative to introduce the `raw` option to `get_storage`.

How about getting the raw storage by accessing the `storage._backend` property?

```python
cached_storage = optuna.storages.get_storage(storage_url)
assert isinstance(cached_storage, _CachedStorage)
storage: Union[RDBStorage, RedisStorage] = cached_storage._backend
```

Reply:
> I think quite a few users want to get raw storage because it significantly
> increases the number of queries. So, I'm a bit conservative to introduce
> the raw option to get_storage.

I've heard that there is a plan to add a raw option to get_storage().
https://github.com/optuna/optuna/pull/3204#discussion_r789365501

Therefore, I have already added this comment.
https://github.com/masap/optuna/blob/a397bb1835754cbc2d510764b12a0a0ca4839762/tests/storages_tests/test_storages.py#L1251-L1255

Could you form a consensus on the implementation policy of get_storage() in optuna maintainer please?

Reply:
Thank you for your comment. It is related to the storage API reconstruction discussed in https://github.com/optuna/optuna/issues/2943. Please give me some more time as I am discussing this with other committers.

Reply:
OK, I will wait for comment.

Reply:
@masap We discussed whether or not to make `_CachedStorage` available for users. The conclusion was that it was too early to release it yet, so we decided not to release it in V3. Related to that, we also decided not to add a "raw" option to `get_storage`.

Reply:
Ok, I removed related comments from `test_store_infinite_values` and `test_fail_stale_trials_raw`.

Reply:
Thanks!